### PR TITLE
[Python] Add missing bits for part 2

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/python-2-a/README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+Besides compressing values, each compression scheme should also support decompressing these compressed values back to their original value. Your job is to add a function named `decompress` that can decompress the compressed `bytes` value to its original UTF-8 `str`:
+
+```python
+>>> decompress(b'\x01a\x02b\x03c\x04d\x0)
+"abbcccddddaaaaaa"
+```
+
+[str]: https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str
+[ascii]: https://en.wikipedia.org/wiki/ASCII
+[nato]: https://en.wikipedia.org/wiki/NATO_phonetic_alphabet#
```
Comment (iHiD) [inline exercises/python-2-a/README.md]:
```suggestion
[nato]: https://en.wikipedia.org/wiki/NATO_phonetic_alphabet
```

File: exercises/python-2-a/README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+[caesar]: https://en.wikipedia.org/wiki/Caesar_cipher
```
Comment (iHiD) [inline exercises/python-2-a/README.md]: This should be RLE I think?

Comment (ErikSchierboom) [reply exercises/python-2-a/README.md]: Yep. Copy-paste error. Updated!

**Comment by ErikSchierboom** [comment]
Thanks for the review!

**Comment by iHiD** [comment]
Thanks for doing it :)

**Pull request merged**